Q/QPALM/build_tarballs.jl

```diff
@@ -65,15 +39,11 @@ platforms = supported_platforms()
 
 # The products that we will ensure are always built
 products = Product[
-                   LibraryProduct("libladel", :libladel),
-                   LibraryProduct("libqpalm", :libqpalm),
+    LibraryProduct("libqpalm_jll", :libqpalm_jll),
```

Comment:
Interesting file name!  I gather this is like a Julia-specific library?

Reply:
Indeed, it includes functions that are only used in the Julia interface. Is this okay, or would it be better to have a Julia-specific library that then depends on a shared `libqpalm` library?

Reply:
We usually tend to have generic libraries, but if you're the maintainer of both the upstream package and the julia interface, I guess the decision is up to you :slightly_smiling_face: